### ISSUE TITLE
Error: No windowing support

### ISSUE THREAD

**Issue by rjpgt** [samizdatco/skia-canvas#197]
I am on Ubuntu 24.10. GPU is integrated Intel UHD Graphics. I get this error when I try to run the 'Rendering to a window' example on the About page:

...node_modules/skia-canvas/lib/classes/gui.js:306
   neon.App.openWindow(JSON.stringify(win.state), core(win.canvas.pages[win.state.page-1]))

Error: No windowing support: Vulkan device was instantiated but unable to render.

I am using node version 22.11.0.

**Comment by samizdatco** [comment]
Apparently several generations of Intel GPUs had buggy MSAA support that would either crash or cause rendering artifacts. As a result, Skia [disallows MSAA](https://issues.chromium.org/issues/40434119) on Intel integrated GPUs even if the device claims it has support. 

Version 2.0.1 now checks with Skia as well as the GPU when creating surfaces (whereas before it was using the GPU's claimed MSAA capabilities and then receiving errors from Skia when it tried to allocate a framebuffer). Note that the 'rocket lake' era GPU I tested had intense visual glitching until I upgraded my Mesa drivers from 24.2.1 to 24.3.1. I used [this PPA](https://launchpad.net/~kisak/+archive/ubuntu/kisak-mesa) to get a copy for my old installation of Ubuntu, but the proper source depends on the distro you're running...
